app/views/autotune/application/google_auth.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-lg-8 col-lg-offset-2">
+      <br>
+      <div class="well">
+        <p>Looks like you haven't logged in with a Google account.</p>
+        <p>As part of adding live preview to Autotune,
+        we now have some blueprints that need to be able to access Google spreadsheet during the build process.
+        Please authenticate with Google so that you can use our live preivew enabled blueprints.</p>
```

Comment:
Don't think we need so much technical detail as to why. Should change it to something like...
"You must link your Google account to Autotune in order to use spreadsheets and other docs in your projects. Please authenticate with Google to link your account to Autotune"